nowplaying/show/client.py

```diff
@@ -34,6 +26,8 @@ class ShowClient:
     Every show has a name, a start and endtime and an optional URL.
     """
 
+    __DEFAULT_SHOW_DURATION = 30  # 30 seconds
```

Comment:
We might move this to config at a later stage. Right now it's being protected by `__` prefixing just to ensure that it doesn't get used all over the place.